db/schema.rb

```diff
@@ -135,6 +135,7 @@
     t.datetime "updated_at",                      null: false
     t.integer  "local_school_id"
     t.boolean  "paid",            default: false
+    t.integer  "workshop_id"
```

Comment:
If you're adding this to registrants, better change up the models as well to reflect this new association